### DISCUSSION TITLE
windows11用vbs,bat等方式隐藏命令窗口后台运行时用户密码错误

### DISCUSSION THREAD

**Discussion by Animism001** [AlistGo/alist#2008] [Invalid question]
windows v3版用vbs启动提示密码错误……
Set ws = CreateObject("Wscript.Shell")
ws.run "alist.exe server",vbhide
用别的隐藏终端命令窗口的方式也是不行
if "%1"=="hide" goto CmdBegin
start mshta vbscript:createobject("wscript.shell").run("""%~0"" hide",0)(window.close)&&exit
:CmdBegin
alist server

alist程序已经添加到系统环境变量里了
我用上面的方法运行alist，用原来的用户名和密码登录提示用户名和密码错误，用admin用户名登录则是密码错误，然后用命令行终端运行alist admin显示的又是原来的用户名与密码……晕了
是不是用ps终端黑窗口运行跟隐藏在后台运行时的初始配置文件位置不同导致的？请问该怎么解决呢，不然我就只能借助第三方隐藏窗口了，但是这样不能开机就自动运行alist server了，每次都要手动开启，救命！！！

**Comment by simear2004** [comment]
一样的问题，还是老老实实手动启动吧。

**Reply by f63531119** [reply]
你自己写的有问题 vbs
Dim ws
Set ws = Wscript.CreateObject("Wscript.Shell")
ws.run "alist.exe server",vbhide
Wscript.quit

bat 用win自带的计划任务

**Reply by jilefo** [reply]
https://www.52pojie.cn/thread-1713978-1-1.html 用这个帖子的工具吧。

**Comment by HCLonely** [comment]
vbs放到alist文件夹

**Comment by DN-NateRiver** [comment]
请问您的问题是否解决？我是WIN10系统，使用相同的vbs代码，启动后一样提醒密码错误，或者”已使用错误密码多次登录，等待后再试“

**Comment by Animism001** [comment]
仔细看我写的内容，解决方案我之前就给出来了哦，就是命令行运行alist server，然后打开web端，将一切设置好之后，把C:\Users\你的用户名这个路径里面alist生成的数据移动到你的alist.exe程序目录下，然后再运行那个VBS，就能用之前设置的用户名/密码登录了

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: ***@***.***>
发送时间: 2022年11月6日 18:59
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [alist-org/alist] windows11用vbs,bat等方式隐藏命令窗口后台运行时用户密码错误 (Discussion #2008)


请问您的问题是否解决？我是WIN10系统，使用相同的vbs代码，启动后一样提醒密码错误，或者”已使用错误密码多次登录，等待后再试“

―

**Comment by DN-NateRiver** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        感谢大佬。
        我这边又遇到了新的问题，能否再次请教您？我这边软件版本：V3，安装位置是D盘。cmd server启动，并且网页端正常登录，设置完成后。C:\users\xxxx这个路径下并不存在alist文件夹或相关文件。
    
        
    
     
    
        
                    
        
    
 

  
    ---- 回复的原邮件 ----
  
  
    
      
        
         发件人 
        
        
            ***@***.***>
            
        
      
      
        
         发送日期 
        
        
        2022年11月6日 20:28
        
      
      
        
         收件人 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         抄送人 
        
        
          
            ***@***.***>
            ,
          
          
            ***@***.***>
            
          
        
      
      
        
         主题 
        
        
              Re: [alist-org/alist] windows11用vbs,bat等方式隐藏命令窗口后台运行时用户密码错误 (Discussion #2008)
        
      
    
  
  
仔细看我写的内容，解决方案我之前就给出来了哦，就是命令行运行alist server，然后打开web端，将一切设置好之后，把C:\Users\你的用户名这个路径里面alist生成的数据移动到你的alist.exe程序目录下，然后再运行那个VBS，就能用之前设置的用户名/密码登录了



从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送



发件人: ***@***.***>

发送时间: 2022年11月6日 18:59

收件人: ***@***.***>

抄送: ***@***.***>; ***@***.***>

主题: Re: [alist-org/alist] windows11用vbs,bat等方式隐藏命令窗口后台运行时用户密码错误 (Discussion #2008)





请问您的问题是否解决？我是WIN10系统，使用相同的vbs代码，启动后一样提醒密码错误，或者”已使用错误密码多次登录，等待后再试“



―

**Comment by Animism001** [comment]
将文件资源管理器显示隐藏文件的选项打开试试，就两个文件夹：data和log，不然就用everything搜索一下用户名下的文件，查找这两个文件夹，win10的我没试过，我的系统是win11，所以可能有所不同，需要你自己找找看。如果用户名路径下找不到，可能是放在.appdata这个文件夹下。我的经验只是告诉大家powershell和vbs两种启动方式，alist生成的数据文件位置不同而已，具体在哪个位置可能因系统而异，或者是因为用了管理员模式？普通模式的ps生成的数据文件就在用户名下的

**Comment by DN-NateRiver** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    非常感谢，
        
    
        我这边data和log两个文件夹在alist安装文件夹中，我在试试。 
        
    
    
     
    
        
                    ***@***.***
        
    
 

  
    ---- 回复的原邮件 ----
  
  
    
      
        
         发件人 
        
        
            ***@***.***>
            
        
      
      
        
         发送日期 
        
        
        2022年11月7日 10:03
        
      
      
        
         收件人 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         抄送人 
        
        
          
            ***@***.***>
            ,
          
          
            ***@***.***>
            
          
        
      
      
        
         主题 
        
        
              Re: [alist-org/alist] windows11用vbs,bat等方式隐藏命令窗口后台运行时用户密码错误 (Discussion #2008)
        
      
    
  
  


将文件资源管理器显示隐藏文件的选项打开试试，就两个文件夹：data和log，不然就用everything搜索一下用户名下的文件，查找这两个文件夹，win10的我没试过，我的系统是win11，所以可能有所不同，需要你自己找找看。如果用户名路径下找不到，可能是放在.appdata这个文件夹下。我的经验只是告诉大家powershell和vbs两种启动方式，alist生成的数据文件位置不同而已，具体在哪个位置可能因系统而异，或者是因为用了管理员模式？普通模式的ps生成的数据文件就在用户名下的

**Comment by Animism001** [comment]
或许你还可以试试把alist程序路径下生成的data和log复制一份到用户名路径下，在运行alist server和alist admin，看看用户密码，然后关闭终端，再用vbs启动alist，用刚刚得到的密码登录试试
